client/src/components/FriendsRecommendations/FriendsRecommendations.stories.tsx

```diff
@@ -6,30 +6,35 @@ import {FriendsRecomendations} from './index';
 
 export const FriendsRecomendationsState = {
     loggedUsername: 'Maks',
+    loadFollow: false,
     friendsRecommendations: {
         loading: false,
         users: [
             {
                 _id: '1f',
                 username: 'Diana',
                 photoPath: 'https://jsgram-post-images.s3.eu-west-2.amazonaws.com/1570605706200',
+                alreadyFollow: false,
```

Comment:
bad name. Usually flag use with prefix is

Reply:
fixed